test/ClangImporter/objc_parse.swift

```diff
@@ -311,10 +311,10 @@ func ivars(_ hive: Hive) {
   hive.queen.description() // expected-error{{value of type 'Hive' has no member 'queen'}}
 }
 
-class NSObjectable : NSObjectProtocol {
-  @objc var description : String { return "" }
-  @objc(conformsToProtocol:) func conforms(to _: Protocol) -> Bool { return false }
-  @objc(isKindOfClass:) func isKind(of aClass: AnyClass) -> Bool { return false }
+class NSObjectable : NSObject {
```

Comment:
I think it'd be more interesting if this didn't subclass anything, but it probably doesn't really matter at this point.